src/main/java/gregtech/GTMod.java

```
@@ -600,7 +600,7 @@ public void onLoadComplete(FMLLoadCompleteEvent aEvent) {
             try {
                 tRunnable.run();
```

Can we please ditch some more of these random Throwable catches, especially the ones around the pre, post etc. Runnables.

I had a bitch of a time the other day trying to figure out why my code was seemingly doing nothing, only to realise that GT5U was silently catching otherwise useful exceptions. These runnables are used in many different places and this makes debugging them much more annoying.